src/main/java/com/dev/cinema/PasswordValidator.java

```diff
@@ -0,0 +1,22 @@
+package com.dev.cinema;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import javax.validation.Constraint;
+import javax.validation.Payload;
+
+@Constraint(validatedBy = PasswordValidatorImpl.class)
+@Target({ ElementType.TYPE })
+@Retention(RetentionPolicy.RUNTIME)
+public @interface PasswordValidator {
+    String fieldMatch() default "verifyPassword";
```

Comment:
what is the name of this field in your UserRequestDto?

Reply:
repeatPassword